### PR TITLE
Honey Bee Pollen Detection

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Honey Bee Pollen Detection

- The aim of this project is to classify the images of Honey bees to detect whether they are carrying pollen grains or not.

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [ ] Contributor

Closes: #418 

### Describe the add-ons or changes you've made

n this project we have first made visualizations of the Bee images dataset using different techniques to understand the correlation, density and other insights of the data.Then we have splitted tbhe whole image set into Training, Validations and Testing set. These sets has been visualized individually in their Original and Greyscale form. Finally, a CNN bianry classification model ahs been build which gives a validation accuracy of 86%. This trained has then been used to predict whether the bee images in the test set are carrying pollen or not.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
PR approved. As I know, it's fab. ✨✨@Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
> PR approved. As I know, it's fab. ✨✨@Soumayan-pal01



> PR approved. As I know, it's fab. ✨✨@Soumayan-pal01

Thank you so much @prathimacode-hub 😋.

**Comment by Soumayan-pal01** [comment]
> PR approved. As I know, it's fab. ✨✨@Soumayan-pal01

Thank you so much @prathimacode-hub 😋.
